.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,3 +26,8 @@ List all PRs related to resolving a ticket. For instance, if you submitted a PR
 - Has server changes (please link here)
 - Has redux changes (please link here)
 - Has mobile changes (please link here)
+
+#### Screenshots
+<!--
+[If the PR includes UI changes, include screenshots/GIFs.]
```

Comment:
```suggestion
If the PR includes UI changes, include screenshots/GIFs.
```

Reply:
Unless there's a reason we want to include inside square brackets

Reply:
@jasonblais No reason. It was just keeping consistent with Mobile PR template (https://github.com/mattermost/mattermost-mobile/blob/master/PULL_REQUEST_TEMPLATE.md) but I forgot it got put into a comment block . I have removed them. Thanks for the catch!